workshop/Lab7/README.md

```diff
@@ -161,7 +186,7 @@ For an application running within a Kubernetes cluster to be able to access an I
 1. Change into the `yaml` directory. apply the `cloudant-ibmcloud.yaml` file.
 
     ```console
-    cd $HOME/kube-storage101/src/yaml
+    cd $HOME/guestbook-nodejs-config/src/yaml
```

Comment:
This path does not exist. should be something like "$HOME/guestbook-nodejs-config/storage/lab7"